mite_selenium/__init__.py

```diff
@@ -9,6 +9,7 @@
 from selenium.webdriver import Remote as SeleniumRemote
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.chrome.options import Options
```

Comment:
Need to find a way to establish is user is using Chrome or Firefox to pull the right Options object